### ISSUE TITLE
DiscordPortable requiring admin priviledge

### ISSUE THREAD

**Issue by jelle619** [daemondevin/DiscordPortable#1]
DiscordPortable requests admin priviledges. However, if this is supposed to be a portable app, shouldn't that be the case? I can't run this on any computer without me having access to the administrator accout.

**Comment by LastChild** [comment]
Same here, kinda makes it useless

**Comment by daemondevin** [comment]
My apologize to the both of you but if you really feel that way @LastChild than don't fucking use it. It's nice to know my work is useless.

@jelle619 I'll fix it when I can but I cannot do anything until I have the free time and I'm in no position to do anything at the moment.

**Comment by jelle619** [comment]
@daemondevin sure! Take your time, no pressure. This is just something I noticed while using it. I thought there was something wrong on my end.

**Comment by daemondevin** [comment]
@jelle619 No.. I'm almost certain there is a minor error within the custom code somewhere. The same problem my DIscord Portable is having is happening with my GitHub Desktop Portable as well.

This is how they're supposed to work; on launch it asks the user for administrative privileges so it can make use of some junction support for faster launch speeds and if the end user selects no than the portable launcher should resort to the old method of moving and/or copying files and folders which takes longer to launch the application.

I'll eventually get around to fixing it. I'm just busy with work now.

**Comment by Aquiliscs** [comment]
don't work

**Comment by Chaython** [comment]
Hello, this seems dead. Is it? Sorry.
Here is an alternative.
https://github.com/portapps/discord-portable
Please review.
If this discord portable is dead, perhaps you could archive/private it, so it's hidden from Github's SEO. - So People don't install it and complain.
